src/internal/components/configurationPanels/GeoPushpinConfigurationPanel.tsx

```diff
@@ -13,11 +27,70 @@ export default class GeoPushpinConfigurationPanel extends ConfigurationPanelCont
         });
     }
 
+    protected getControlProperties() {
+        const props = this.props;
+        const groupNearbyPoints = get(props, "properties.controls.groupNearbyPoints", true);
+
+        return {
+            groupNearbyPoints,
+        };
+    }
+
+    protected renderPointsSection() {
+        const { groupNearbyPoints } = this.getControlProperties();
+
+        const { properties, propertiesMeta, pushData, mdObject } = this.props;
+        const controlsDisabled = this.isControlDisabled();
+        const isClusteringDisabled = controlsDisabled || hasMeasures(mdObject) || hasSegment(mdObject);
+        const isPuspinSizeControllDisabled = controlsDisabled || !hasSizeMeasure(mdObject);
+        return (
+            <ConfigSection
+                id="points_section"
+                title="properties.points.title"
+                propertiesMeta={propertiesMeta}
+                properties={properties}
+                pushData={pushData}
+            >
+                <CheckboxControl
+                    valuePath="groupNearbyPoints"
+                    labelText="properties.points.groupNearbyPoints"
+                    properties={properties}
+                    checked={groupNearbyPoints}
+                    disabled={isClusteringDisabled}
+                    pushData={pushData}
+                />
+                <PushpinSizeControl
+                    properties={properties}
+                    disabled={isPuspinSizeControllDisabled}
+                    pushData={pushData}
+                />
+            </ConfigSection>
+        );
+    }
+
     protected isControlDisabled() {
-        return true;
+        const { mdObject, isError, isLoading } = this.props;
+        return !hasLocation(mdObject) || isError || isLoading;
+    }
+
+    protected getBubbleClassNames() {
```

Comment:
Apart from overwritting base methods, other functions should be defined `private`